pattern-hues.lic

```diff
@@ -56,14 +55,14 @@ class PatternHues
       /You should stop practicing/,
       /You find it hard to concentrate on anything/
     ]
-    scripts_to_unpause = DRC.smart_pause_all
-    bput("prep cantrip pattern hues", "You are now prepared to cast the Pattern Hues cantrip.","You already have", "You should stop practicing")
+    pattern_hues_pause_list ||= DRC.safe_pause_list
+    DRC.bput("prep cantrip pattern hues", "You are now prepared to cast the Pattern Hues cantrip.","You already have", "You should stop practicing")
     if @settings.pattern_hues['gesture'] == "random"
-      bput("gesture #{@adj.sample} #{@color.sample}", *match_messages)
+      DRC.bput("gesture #{@adj.sample} #{@color.sample}", *match_messages)
     else
-      bput("gesture #{@settings.pattern_hues['gesture']}", *match_messages)
+      DRC.bput("gesture #{@settings.pattern_hues['gesture']}", *match_messages)
     end
-    DRC.unpause_all_list(scripts_to_unpause)
+    DRC.safe_unpause_list(pattern_hues_pause_list)
```

Comment:
The new method doesn't work like this, and you have to have a check for the return value of the method.

See https://github.com/rpherbig/dr-scripts/pull/6044

Reply:
DRC.safe_unpause_list(scripts_to_unpause) is in 6044.

Reply:
I'm sorry, I misread.